deploy/olm-catalog/service-telemetry-operator/metadata/annotations.yaml

```diff
@@ -1,6 +1,6 @@
 annotations:
   operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: latest
+  operators.operatorframework.io.bundle.channels.v1: stable,latest
```

Comment:
@leifmadsen I used the bundle update command you showed me - is this the right channels setting (I haven't looked this up yet to be 100% sure I know what it means)

Reply:
oh right, sorry, actually you want `--channels latest` and `--default-channel stable`.

* --channels is a list of channels this bundle can be a part of -- we only want HEAD to be in `latest` channel
* --default-channel defines the bundles default preferred channel -- we want this as stable so that we don't auto-install `latest` channel